### ISSUE TITLE
Missing Vagrant VMware Utility

### ISSUE THREAD

**Issue by U1E9E** [hashicorp/homebrew-tap#195]
Unfortunately, the [Vagrant VMware utility](https://developer.hashicorp.com/vagrant/downloads/vmware) is missing here.

**Comment by mdeggies** [comment]
We'll add this shortly. Thanks for the note!

**Comment by Sn0wCrack** [comment]
Is there chance of this getting added to this tap still?

I noticed a new version came out with ARM64 support, I imagine that was the blocker for this being added?

**Comment by der-eismann** [comment]
Agree, additionally the current situation is super confusing. In the [docs for vmware](https://developer.hashicorp.com/vagrant/docs/providers/vmware/installation) it's linking to this page:
https://developer.hashicorp.com/vagrant/install/vmware

There you have - as always - package manager and binary downloads, however the former is just the `hashicorp/tap/hashicorp-vagrant` cask and not the wanted utility.
